src/cargo/core/summary.rs

```diff
@@ -227,4 +230,20 @@ impl FeatureValue {
     }
 }
 
+impl Serialize for FeatureValue {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        use self::FeatureValue::*;
+        match *self {
```

Comment:
shorter, but less performance code is: `serializer.serialize_str(*self.to_string())`. But this works too.

Reply:
Ok, then this looks good to me. 👍